### PR TITLE
Form label less emphasis

### PR DESCRIPTION
**Pull request by stewartyu**
### Issues

Adding a `.form-label--less-emphasis` modifier.

screen shot 2016-10-24 at 1 50 30 pm

### CODE REVIEW

File: src/forms/_base.scss [hunk 1]
```diff
@@ -184,18 +184,27 @@ select {
...
-      display: block;
-      margin-bottom: ms(-6);
-      color: color(text);
+  & + .form-input {
+    margin-top: ms(-6);
+  }
 
-      & + .form-input {
-        margin-top: ms(-6);
+  &.form-label--mixed-case {
```
Comment (robwierzbowski) [inline src/forms/_base.scss]: It seems like the purpose of this is to be lower in hierarchy than a title case label. Can the modifier name reflect that?

File: src/forms/_base.scss [hunk 2]
```diff
@@ -226,6 +235,11 @@ select {
     transform: translateY(1rem) scale(1);
     transition: transform 0.25s ease-out;
     transform-origin: 0 0;
+    text-transform: none;
+
+    + .form-input {
```
Comment (robwierzbowski) [inline src/forms/_base.scss]: Can you use the `&` here, as on line 195?

Comment (stewartyu) [reply src/forms/_base.scss]: I'd actually prefer to remove the `&` on line 195 as it's not actually necessary. I prefer only using the `&` when you need to change the default nesting order, as in `.foo &`. Thoughts?

File: src/forms/_base.scss (hunk 1 above)
Comment (stewartyu) [reply src/forms/_base.scss]: That's fair - any thoughts? Some of my original thoughts were `less-emphasis`, `level-2`, `lower` ... do we like any of these better?

File: src/forms/_base.scss (hunk 2 above)
Comment (robwierzbowski) [reply src/forms/_base.scss]: I think it's standard in the Casper codebase to use the `&` explicitly except when doing simple descendent nesting. I would be down to talk about a global refactor.

File: src/forms/_base.scss (hunk 1 above)
Comment (robwierzbowski) [reply src/forms/_base.scss]: `subheading`... I like all of these better than mixed case. I leave it up to you.

**Comment by stewartyu** [comment]
@robwierzbowski Comments addressed!

**Comment by robwierzbowski** [comment]
DTM!

**Pull request merged**